experimental/single_number_formatter/src/currency_formatter.rs

```diff
@@ -0,0 +1,40 @@
+
+// FormattingData:
+// .  Yes, we could merge them.
+
+struct CurrencyFormatter {
+    options: CurrencyFormatterOptions,
+    data: DataPayload<FormattingData>,
+}
+
+
+struct CurrencyFormatterOptions {
+    /// Determine the formatting width.
+    /// For example for meter (FULL: 2 US Dollars) and (NARROW: 2$).
+    pub width: Width,
+
+    /// Determine the single unit which can be SingleMeasurementUnit or SingleCurrencyUnit
```

Comment:
measurement?

Reply:
removed, i wanted the boiler-print for the next pr.